libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -131,11 +131,12 @@ class AP_InertialSensor_Backend
         DEVTYPE_INS_ICM42605 = 0x35,
         DEVTYPE_INS_ICM40605 = 0x36,
         DEVTYPE_INS_IIM42652 = 0x37,
-        DEVTYPE_BMI270       = 0x38,
-        DEVTYPE_INS_BMI085   = 0x39,
-        DEVTYPE_INS_ICM42670 = 0x3A,
-        DEVTYPE_INS_ICM45686 = 0x3B,
-        DEVTYPE_INS_SCHA63T  = 0x3C,
+        DEVTYPE_INS_IIM42653 = 0x38,
```

Comment:
I don't think we can renumber these - they're logged and tools like `decode_devid.py` would insta-break.